### PR TITLE
docs: Add image and adjust structure to collection view headers [JOB-111089]

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Jobber's design system docs should feel like Jobber, and bring some of our brand _vibes_ into our design guidance.

Imagery like this is commonplace in our marketing site and materials:
image

### Changed

- Styling of collection page headers
  - now accepts a background image
  - uses paper-rip styling to mask the bottom of the image
image

## Testing

View any collection page (Home, Components, Design)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/site/src/pages/DesignPage.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ export const DesignPage = () => {
         header: {
           title: "Design",
           body: "Design elements and concepts that apply across Jobber's ecosystem",
+          imageURL:
+            "https://images.unsplash.com/photo-1482731215275-a1f151646268?q=80&w=1470&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
```
Comment (chris-at-jobber) [inline packages/site/src/pages/DesignPage.tsx:12]: Placeholder image to validate that a non-default image can be passed in 
image

File: packages/site/src/main.css [hunk 2]
```diff
@@ -129,4 +129,33 @@ section[role="tabpanel"]>div>h1:first-of-type {
 iframe {
   width: 100%;
   min-height: 200px;
+}
+
+.headerBlock {
+  background-image: url("../public/img_collage.jpg");
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:135]: fallback image in the event no prop is passed in

File: packages/site/src/pages/DesignPage.tsx [hunk 3]
```diff
@@ -7,7 +7,8 @@ export const DesignPage = () => {
       structure={{
         header: {
           title: "Design",
-          body: "Design elements and concepts that apply across Jobber's ecosystem",
+          body: "The foundation of Jobber's look and feel",
```
Comment (chris-at-jobber) [inline packages/site/src/pages/DesignPage.tsx:10]: image

File: packages/site/src/pages/HomePage.tsx [hunk 4]
```diff
@@ -9,6 +9,7 @@ export const HomePage = () => {
           body: "Design and build consumer-grade experiences with Jobber's design system",
           ctaLabel: "Get Started",
           to: "/content/design/welcome-guide",
+          imageURL: "../public/img_collage.jpg",
```
Comment (chris-at-jobber) [inline packages/site/src/pages/HomePage.tsx:12]: image

**Review by Beccaneer — APPROVED** [review]

**Pull request merged**